### ISSUE TITLE
Replace coming soon image gallery images

### ISSUE THREAD

**Issue by ilyasfoo** [woothemes/woocommerce#54036]
### Issue description

Follow-up to https://github.com/woocommerce/woocommerce/pull/53910#discussion_r1897612216

> The better solution might be to provide image without any border radius, so it will not have that odd white borders when users remove border radius.

![Image](https://github.com/user-attachments/assets/e298afa9-14bc-4c83-bb7b-306e867f99f3)

#### To reproduce:

1. Enable the feature flag `coming-soon-newsletter-template`
2. Go to `Appearance > Editor > Templates` and look for coming soon template
3. Select image gallery pattern under `Design` panel
4. Change background color to something non-white
5. Select any of the images and remove border radius
6. Observe the white borders on images

### Acceptance criteria

1. Export images without any rounded borders from Figma
2. Optimize images sizes

**Comment by chihsuan** [comment]
> The better solution might be to provide image without any border radius, so it will not have that odd white borders when users remove border radius.

![Image](https://github.com/user-attachments/assets/e298afa9-14bc-4c83-bb7b-306e867f99f3)

@adrianduffell  @rjchow @ilyasfoo I was trying to update the gallery images on the template as previously discussed [here](https://github.com/woocommerce/woocommerce/pull/53910#discussion_r1897612216), but I found that the style/dimension of the source images does not match the images on the design. (0BWMerqRR1f468EwC7g5tg-fi-3652_27792). In other words, I couldn't get the exact gallery images without any rounded borders from Figma.

We'll need to ask Veronica for help getting the proper images. However, I wonder if it's necessary to update the photos. If they remove the rounded borders, I think they would also use their images. It doesn't make sense that users want to remove the rounded borders without changing the default images. 🤔 

What do you think?

**Comment by ilyasfoo** [comment]
> However, I wonder if it's necessary to update the photos. If they remove the rounded borders, I think they would also use their images. It doesn't make sense that users want to remove the rounded borders without changing the default images. 🤔

@chihsuan Good point. IMO, this is a nice-to-have. If the effort requires designer to redo images, perhaps it might not be worth it. If there's no objection, I think we could close this issue.
